bridgeservice/bridge.go

```diff
@@ -41,14 +42,15 @@ const (
 	BridgeV1Prefix = "/bridge/v1"
 	meterName      = "github.com/agglayer/aggkit/bridgeservice"
 
-	networkIDParam    = "network_id"
-	networkIDsParam   = "network_ids"
-	pageNumberParam   = "page_number"
-	pageSizeParam     = "page_size"
-	depositCountParam = "deposit_count"
-	fromAddressParam  = "from_address"
-	leafIndexParam    = "leaf_index"
-	globalIndexParam  = "global_index"
+	networkIDParam      = "network_id"
+	networkIDsParam     = "network_ids"
+	pageNumberParam     = "page_number"
+	pageSizeParam       = "page_size"
+	depositCountParam   = "deposit_count"
+	fromAddressParam    = "from_address"
+	leafIndexParam      = "leaf_index"
+	globalIndexParam    = "global_index"
+	populateProofsParam = "populate_proofs"
```

Comment:
we can also convert this to `fullResponse` in which we will get all claims field and not just this data.

Reply:
then rename it to full_response/include_all_fields, something like that. not restrict to specific fields